src/patternfly/components/Divider/divider.scss

```diff
@@ -1,9 +1,67 @@
+$pf-c-divider--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl", "2xl");
+$pf-c-divider--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl", "3xl");
+
 .pf-c-divider {
+  // update during breaking change
   --pf-c-divider--Height: var(--pf-global--BorderWidth--sm);
   --pf-c-divider--BackgroundColor: var(--pf-global--BorderColor--100);
+  --pf-c-divider--after--Height: var(--pf-c-divider--Height);
+  --pf-c-divider--after--BackgroundColor: var(--pf-c-divider--BackgroundColor);
+
+  // end - update during breaking change
+
+  --pf-c-divider--m-vertical--Width: var(--pf-global--BorderWidth--sm);
 
-  height: var(--pf-c-divider--Height);
-  background-color: var(--pf-c-divider--BackgroundColor);
+  // After
+  --pf-c-divider--after--FlexBasis: 100%;
+  --pf-c-divider--after--Inset: 0%;
+
+  display: flex;
+  align-items: center;
+  align-self: stretch;
+  justify-content: center;
+  width: 100%;
   border: 0; // removes the default border styling on an hr
-  flex-shrink: 0;
+
+  &::after {
+    flex-basis: calc(var(--pf-c-divider--after--FlexBasis) - calc(var(--pf-c-divider--after--Inset) * 2));
+    align-self: stretch;
+    height: var(--pf-c-divider--after--Height);
+    content: "";
+    background-color: var(--pf-c-divider--after--BackgroundColor);
+    justify-self: center;
+  }
+
+  &.pf-m-vertical {
+    display: inline-flex;
+    flex-direction: column;
+    width: var(--pf-c-divider--m-vertical--Width);
```

Comment:
why does this have a width set here and on the pseudo class

Reply:
Good catch, it was originally to unset `width: 100%`, but I don't think we need that either

Reply:
Update: we need `width: 100%`. Vertical width is now `auto`